CONTRIBUTING.md

```diff
@@ -62,32 +62,40 @@ To enable additional checks (including `-Werror` / `/WX` compilation flag), swit
 ["CMake standard options"](./README.md#cmake-standard-options) section in the top-level Readme.
 
 ### Code style
-
-We use `clang-format` to verify and apply code style changes. To see all rules we require,
-please take a look at `.clang-format` file in the root directory of this repository.
+We use `clang-format` to verify and apply code style changes to source files. 
+To see all rules we require, please take a look at `.clang-format` file in the 
+root directory of this repository. Similarly, we use `cmake-format` tool and
+`.cmake-format` file to verify and apply code style changes to CMake files.
 
 To enable code style checks and re-formatting, CMake option `UMF_FORMAT_CODE_STYLE` has to
-be switched on. You'll then have two additional CMake targets available.
+be switched on. You'll then have additional CMake targets available.
 
 To verify correct coding style of your changes execute (assuming `build` is your build directory):
 
 ```bash
 $ cmake -B build -DUMF_FORMAT_CODE_STYLE=ON
-$ cmake --build build --target clang-format-check
+$ cmake --build build --target format-checks
 ```
 
-We run this check in our Continuous Integration (CI). So, if any issues were found,
+We run these checks in our Continuous Integration (CI). So, if any issues were found,
 the Pull Request will be blocked from merging. To apply proper formatting (meaning,
-to fix the issues) execute the second available CMake target - run a command:
+to fix the issues) you can use the convenience target provided for applying formats:
 
 ```bash
-$ cmake --build build --target clang-format-apply
+$ cmake --build build --target format-apply
 
 # Remember to review introduced changes
 ```
 
-**NOTE**: We use specific clang-format version - **15.0** is required. It can be installed,
-e.g., with command: `python -m pip install clang-format==15.0.7`.
+If you wish to use only `clang-format` or only `cmake-format`, you can execute the corresponding
+`clang-format-check` and `clang-format-apply` for source files, or `cmake-format-check` and
+`cmake-format-apply` for CMake files, respectively.
+
+**NOTE**: We use specific versions of formatting tools to ensure consistency across the project. The required versions are:
+- clang-format version **15.0**, which can be installed with the command: `python -m pip install clang-format==15.0.7`.
+- cmake-format version **0.6.13**, which can be installed with the command: `python -m pip install cmake-format==0.6.13`.
```

Comment:
I believe you can just write `- cmake-format version **0.6** ...` patch version should not break compatibility (I don't talk about installation part - it should stay with patch version)

Reply:
Done